docs/docs/reference.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: Reference
 id: reference
-layout: docs
 description: Secretless Broker Documentation
 permalink: docs/reference.html
+redirect_to: https://docs.secretless.io/Latest/en/Content/Resources/_TopNav/cc_Home.htm
```

Comment:
I hope to have an updated link for this that points to Fundamentals > Overview

Reply:
:)

Reply:
https://docs.secretless.io/Latest/en/Content/Overview/how_it_works.htm